Automattic/wp-calypso
# Remove address and phone number steps from signup

#### Changes proposed in this Pull Request

This PR removes the address and phone number steps from our onboarding flow. 

#### Testing instructions

1. Create a new site by visiting `/start/onboarding`.
2. Pick Business on the site type step. 
3. After entering a title, you should see the domain step next.



cc @michaeldcain
